test/setup.js

```diff
@@ -1,50 +0,0 @@
-const fs = require('fs');
-
-require('source-map-support').install();
-
-process.env.TEST = 'true';
-
-require.extensions['.js'] = function (module, filename) {
-	const exports = [];
-
-	let code = fs
-		.readFileSync(filename, 'utf-8')
-		.replace(/^import \* as (\S+) from ['"]([^'"]+)['"];?/gm, 'var $1 = require("$2");')
-		.replace(/^import {([^}]+)} from ['"](.+)['"];?/gm, 'var {$1} = require("$2");')
```

Comment:
Look away, you didn't see anything.

![image](https://github.com/sveltejs/svelte/assets/50981692/7e06944f-bb93-4bea-8f77-6b5482a9ae75)